implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs

```diff
@@ -393,10 +385,63 @@ impl NodePaths {
     }
 }
 
+mod backwards_compatibility {
+    use super::*;
+
+    #[derive(Deserialize, Debug, Clone)]
+    #[serde(untagged)]
+    pub(super) enum NodeConfigs {
+        V1(NodeConfigV1),
+        V2(NodeConfig),
+    }
+
+    #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
+    pub(super) struct NodeConfigV1 {
+        #[serde(flatten)]
+        pub setup: NodeSetupConfigV1,
+        #[serde(skip)]
+        pub version: ConfigVersion,
+        #[serde(skip)]
+        pub default_vault: PathBuf,
+        #[serde(skip)]
+        pub default_identity: PathBuf,
+    }
+
+    #[derive(Deserialize, Debug, Clone)]
+    #[serde(untagged)]
+    pub(super) enum NodeSetupConfigs {
+        V1(NodeSetupConfigV1),
+        V2(NodeSetupConfig),
+    }
+
+    #[derive(Serialize, Deserialize, Debug, Clone, Default, Eq, PartialEq)]
+    pub(super) struct NodeSetupConfigV1 {
+        pub verbose: u8,
+        #[serde(default)]
+        pub disable_file_logging: bool,
+
+        /// This flag is used to determine how the node status should be
+        /// displayed in print_query_status.
+        /// The field might be missing in previous configuration files, hence it is an Option
+        pub authority_node: Option<bool>,
+        pub project: Option<ProjectLookup>,
+        pub transports: HashSet<CreateTransportJson>,
+    }
+
+    #[cfg(test)]
+    impl NodeSetupConfigV1 {
```

Comment:
this method should be unused

Reply:
It's kept only for testing purposes, so we don't have to remove old tests:
- https://github.com/build-trust/ockam/blob/895630c07c9a24aaa8d43a01e84042a4bf7786a1/implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs#L581
- https://github.com/build-trust/ockam/blob/895630c07c9a24aaa8d43a01e84042a4bf7786a1/implementations/rust/ockam/ockam_api/src/cli_state/nodes.rs#L603